src-vue/src/views/DeveloperView.vue

```diff
@@ -74,6 +74,10 @@
                 Get installed mods
             </el-button>
 
+            <el-button type="primary" @click="getAvailableProfiles">
```

Comment:
Is this of any use?

Reply:
Whoops, no. Good catch. Was some leftover code. Removed with 2000de537d17e9b547cdd7d8466d7b863e1bfbb9